tests/test_modeling_vision_text_dual_encoder.py

```diff
@@ -0,0 +1,437 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch VisionTextDualEncoder model. """
+
+
+import collections
+import tempfile
+import unittest
+
+import numpy as np
+
+from transformers import is_torch_available
+from transformers.testing_utils import require_torch, slow, torch_device
+
+from .test_modeling_bert import BertModelTester
+from .test_modeling_clip import CLIPVisionModelTester
+from .test_modeling_common import floats_tensor, ids_tensor, random_attention_mask
+from .test_modeling_deit import DeiTModelTester
+from .test_modeling_roberta import RobertaModelTester
+from .test_modeling_vit import ViTModelTester
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        BertModel,
+        CLIPVisionModel,
+        DeiTModel,
+        RobertaModel,
+        VisionTextDualEncoderConfig,
+        VisionTextDualEncoderModel,
+        ViTModel,
+    )
+
+
```

Comment:
We need a test here that makes sure PT=>Flax and Flax=>PT conversion works correctly

Reply:
(in addition to the tests added to Flax) to make sure the test fetcher doesn't miss a test